MapboxCoreNavigationTests/Heading.swift

```diff
@@ -0,0 +1,35 @@
+import CoreLocation
+
+class Heading: CLHeading {
+	
+	private var _heading: CLLocationDirection
+	private var _accuracy: CLLocationDirection
+	
+	init(heading: CLLocationDirection, accuracy: CLLocationDirection) {
+		self._heading = heading
+		self._accuracy = accuracy
+		super.init()
+	}
+	
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	open override var trueHeading: CLLocationDirection {
+		get {
+			return _heading
+		}
+		set {
+			_heading = newValue
+		}
+	}
+	
+	open override var headingAccuracy: CLLocationDirection {
+		get {
+			return _accuracy
+		}
+		set {
+			_accuracy = newValue
+		}
+	}
+}
```

Comment:
Please stick to the established indentation style throughout the codebase in all files. Consistent indentation improves readability and ensures professionalism.

Reply:
Sure, if you tell me which indentation you mean, looking through the codebase I see different styles. I don't mind fixing it all to be consistent, however we should agree on one style first.

For example MLNMapView uses the same style
![Bildschirmfoto 2024-04-11 um 11 12 21](https://github.com/maplibre/maplibre-navigation-ios/assets/18739004/13ba9b5d-4046-442e-9806-b157270b2dbe)

Reply:
The issue is that there is no convention ;)
But this will be fixed with the formatter.